### PR TITLE
Pr: 4th shot recoveries click;5th shot confirmed deaths click

### PR DESCRIPTION
**Pull request by muamichali**
Tested Locally. ID seems static on multiple browsers on my machine. 🙏

### CODE REVIEW

File: core_screenshot_config.yaml [hunk 1]
```diff
@@ -184,6 +184,11 @@ quaternary:
     overseerScript: page.manualWait(); await page.waitForDelay(60000); page.done();
     message: wait for IA quaternary
 
+  PR:
+    overseerScript: await page.waitForNavigation({waitUntil:"domcontentloaded"}); await page.waitForSelector("#ember906");page.click("#ember906");page.done();
```
Comment (julia326) [inline core_screenshot_config.yaml]:
```suggestion
    overseerScript: >
      await page.waitForNavigation({waitUntil:"domcontentloaded"});
      await page.waitForSelector("#ember906");
      page.click("#ember906");
      page.done();
```

Comment (julia326) [reply core_screenshot_config.yaml]: thank you! Just nit, for readability I've been trying to keep these config files to an easily viewable width so just adding some newlines here

Comment (muamichali) [reply core_screenshot_config.yaml]: of course! fixed

File: core_screenshot_config.yaml [hunk 2]
```diff
@@ -194,3 +197,11 @@ quinary:
   IA:
     overseerScript: page.manualWait(); await page.waitForDelay(60000); page.done();
     message: wait for IA quinary
+
+  PR:
+    overseerScript: 
+      await page.waitForNavigation({waitUntil:"domcontentloaded"}); 
+      await page.waitForSelector("#ember772");
+      page.click("#ember772");page.done();
```
Comment (muamichali) [inline core_screenshot_config.yaml]: oops missed this newsline hang on

File: core_screenshot_config.yaml [hunk 3]
```diff
@@ -184,8 +184,26 @@ quaternary:
...
     overseerScript: page.manualWait(); await page.waitForDelay(60000); page.done();
     message: wait for IA quinary
+
+  PR:
+    overseerScript: 
+      await page.waitForNavigation({waitUntil:"domcontentloaded"}); 
+      await page.waitForSelector("#ember772");
+      page.click("#ember772");
+      page.done();
+    message: click button for PR confirmed Deaths ("Muertes Confirmades")
```
Comment (julia326) [inline core_screenshot_config.yaml:208]: thank you!

**Pull request merged**